bindings/python/PythonLoader.cpp

```diff
@@ -0,0 +1,129 @@
+#include <Python.h>
+#include "gameobject.h"
+#include "swigruntime.py.h"
+#include "scripts/PythonLoader.h"
+#include "script.h"
+#include <string>
+#include <algorithm>
+using std::string;
+using nautical::GameObject;
+using nautical::script::Script;
+
+namespace
+{
+    void addPythonModulePath(string p)
+    {
+        static char pthc[5] = "path";
+        PyObject *sys_path = PySys_GetObject(pthc);
```

Comment:
It's the function we would use from C++, yes. If we figure out a good way to add it to the API, then fine with me. From Python, you can just edit sys.path directly.